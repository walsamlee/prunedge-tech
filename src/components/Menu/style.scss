.menu {
  display: flex;
  flex-direction: column;
  margin: 20px 0;

  .item {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    height: 35px;

    .badge {
      width: 20px;
      height: 15px;
      margin: 10px 48px 10px 0;
      background: #C4C4C4;
      border-radius: 7px;
      text-align: center;

      p {
        margin: 0;
        padding: 1px 0;
        font-size: 10px;
        color: #ffffff;
      }
    }
  }

  .filler {
    height: 370px;
  }

  a {
    display: block;
    padding: 10px 0;
    font-size: 12px;
    line-height: 14px;
    color: #8A94A6;
    text-decoration: none;
    border-left: 3px solid #ffffff;

    i {
      padding: 0 10px 0 20px;
    }
  }

  .active {
    color: #43536F;
    font-weight: bold;
    border-left: 3px solid #0C66FF;
  }

  .logout {
    color: #53627C;

    i {
      color: #8A94A6;
    }
  }
}