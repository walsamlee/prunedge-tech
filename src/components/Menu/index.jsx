// react
import React from 'react';

// style
import './style.scss';

const Menu = () => (
  <div className="menu">
    <div className="item">
      <a className="active" href="/">
        <i className="fas fa-home"></i> Dashboard
      </a>
    </div>
    <div className="item">
      <a href="/">
        <i className="fas fa-bookmark"></i> My Document
      </a>
    </div>
    <div className="item">
      <a href="/">
        <i className="fas fa-bell"></i> Activity
      </a> 
      <div className="badge">
        <p>12</p>
      </div>
    </div>
    <div className="item">
      <a href="/">
        <i className="fas fa-user"></i> Account
      </a>
    </div>
    <div className="filler"></div>
    <div className="item">
      <a className="logout" href="/">
        <i class="fas fa-arrow-circle-left"></i> Log Out
      </a>
    </div>
  </div>
);

export default Menu;
