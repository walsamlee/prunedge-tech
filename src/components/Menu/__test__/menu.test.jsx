// react
import React from 'react';

// third-party
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// component
import Menu from '../';

configure({ adapter: new Adapter() });

describe('Menu component', () => {
  it('should render Menu', () => {
    const wrapper = shallow(
      <Menu />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
