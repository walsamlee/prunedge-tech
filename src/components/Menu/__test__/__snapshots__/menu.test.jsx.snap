// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component should render Menu 1`] = `ShallowWrapper {}`;
